exonum-java-binding/common/src/main/java/com/exonum/binding/common/proofs/map/UncheckedFlatMapProof.java

```diff
@@ -51,6 +52,13 @@
 
   private final List<ByteString> missingKeys;
 
+  @VisibleForTesting
+  static final byte BLOB_PREFIX = 0x00;
+  @VisibleForTesting
+  static final byte MAP_NODE_PREFIX = 0x03;
+  @VisibleForTesting
+  static final byte MAP_BRANCH_NODE_PREFIX = 0x04;
```

Comment:
These names are misleading: _branch_ prefix is applied to both _leaf_ and branch nodes; whilst _node_ prefix is actually applied to compute the map hash (= it is not applied to nodes, but to 'merkle root').

MAP_NODE_PREFIX -> MAP_OBJECT_PREFIX/MAP_ROOT_PREFIX and 
MAP_BRANCH_NODE_PREFIX -> MAP_NODE_PREFIX?

I also suggest improving the naming upstream (cc @pavel-mukhanov , @aleksuss ).